src/views/imaged-object-editor/imaged-object-editor.vue

```diff
@@ -355,17 +357,26 @@ export default class ImagedObjectEditor
 
         return true;
     }
+
     public get canCreate(): boolean {
         return this.newArtefactName.trim().length > 0;
     }
-   public newModalShown() {
+
+    public newModalShown() {
         // this.waiting = true;
         (this.$refs.newArtefactName as any).focus();
     }
-    private async mounted() {
+
+   // moved code to created()
```

Comment:
In this case this was unnecessary - we've already handled the race condition with the `waiting` flag.